Misc100

morebrownies opened the pull request:
**Changes**
- Changed Mac tray icon
- Changed Windows tray icon
- Added tooltip to long listings in the discover section (more than 30 chars)
- Blur background when transaction modal is opened now (required me to move the modal element to index.html)
- Prevent body scroll when transaction modal is open
- Cleaned up the double cell border issue in the store listing view
- Prevent more than two decimal places in the fee input (not sure how great my JS is for this)
- Darkened tooltips to prevent collisions with body content

ilxwolf committed:
prevent more than 2 decimal places in fee input

ilxwolf committed:
added tooltip to title

ilxwolf committed:
change

ilxwolf committed:
Merge branch 'misc99' into misc100

ilxwolf committed:
merge misc99 changes

ilxwolf committed:
better mac tray icons, low res and retina

ilxwolf committed:
windows and mac tray icons ready. Brian needs to put in the logic to display them based on the user's os

ilxwolf committed:
fixes js issue

ilxwolf committed:
transaction changes

morebrownies commented:
@jjeffryes ready for review

jjeffryes commented on js/views/transactionsVw.js:
```diff
@@ -229,7 +229,7 @@ module.exports = baseVw.extend({
       status: options.status,
       serverUrl: this.serverUrl,
-      parentEl: this.$el.find('.js-transactionModalHolder'),
+      parentEl: $('.js-transactionModalHolder'),
```
Why move this to index? It should stay in the transaction view, that way if you navigate away from that view it's automatically removed. 

Are there other modals or elements we want the transaction modal to be layered on top of?

jjeffryes commented on js/views/pageNavVw.js:
```diff
@@ -350,18 +350,19 @@ module.exports = Backbone.View.extend({
     // display the modal
-    $('.js-aboutModal').removeClass('hide');
+    $('.js-aboutModal').removeClass('fadeOut').addClass('fadeIn');
   },
```
We'll have to watch this, previously the fadeIn/fadeOut classes caused problems. 

It should also just need one class, fadeOut, removing it should do all you need.

morebrownies replied:
I was trying to get the modal to fadeIn like the transaction modal, but didn't have much luck.

jjeffryes replied:
I can change it to fade in like the other modal.

morebrownies replied:
It would be nice to get them all working the same way if it's relatively simple.

jjeffryes committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into misc100

jjeffryes committed:
fixes

- fade in/out about and support
- clean up transaction modal on navigation
- remove old about view code
